schema/schema.go

```diff
@@ -80,6 +80,7 @@ type Ignored struct {
 	ForeignKey bool
 }
 
+// Print returns column type to print the fields of struct Type.
```

Comment:
How about:

// Print converts ty to a string suitable for printing.

Reply:
Good, I think my comment is a bit confusing. Updated!